code/modules/pool/components/swimming.dm

```diff
@@ -130,7 +130,7 @@
 		victim.emote("cough")
```

Comment:
Dead people don't cough or gasp either. In fact, they shouldn't even process drown ticks. Also, do `victim.stat`, crit and unconscious shouldn't gasp/fall unconcious either.